db/migrate/20220127002337_create_reactions.rb

```diff
@@ -0,0 +1,13 @@
+class CreateReactions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reactions do |t|
+      t.boolean :statement
```

Comment:
Is it posible for a reaction record not o have a `statement`?
If not, then probably there should be a valuation at db level as well

Reply:
yes, it's possible for instance, I can have this 

| id_user | curiosity_id | statement |
| ----------- | ----------- | ----------- |
| 1 | 2 | niil |
| 1 | 3 | true |

Reply:
Having `statement` as nil...what does the record mean?

Reply:
Well if a record is nil means that the user delete the true or false actions and now the record doesnt have either or is it a better idea deleted it?  🤔